### PR TITLE
[MIG] purchase_open_qty: Migrate to version 17.0

### PR DESCRIPTION
**Pull request by sonhd91**
Ref: BSRD-710

### CODE REVIEW

File: purchase_open_qty/init_hook.py [hunk 1]
```diff
@@ -7,40 +7,40 @@
...
         """SELECT column_name
     FROM information_schema.columns
     WHERE table_name='purchase_order_line' AND
     column_name='qty_to_receive'"""
     )
-    if not cr.fetchone():
-        logger.info("Creating field qty_to_receive on purchase_order_line")
-        cr.execute(
+    if not env.cr.fetchone():
+        logger.info("enveating field qty_to_receive on purchase_order_line")
```
Comment (gurneyalex) [inline purchase_open_qty/init_hook.py]: one "replace all cr->env" too much? :-)

Comment (gurneyalex) [reply purchase_open_qty/init_hook.py]:
```suggestion
        logger.info("Creating field qty_to_receive on purchase_order_line")
```

File: purchase_open_qty/init_hook.py [hunk 2]
```diff
@@ -7,40 +7,40 @@
...
         """SELECT column_name
     FROM information_schema.columns
     WHERE table_name='purchase_order_line' AND
     column_name='qty_to_invoice'"""
     )
-    if not cr.fetchone():
-        logger.info("Creating field qty_to_invoice on purchase_order_line")
-        cr.execute(
+    if not env.cr.fetchone():
+        logger.info("enveating field qty_to_invoice on purchase_order_line")
```
Comment (gurneyalex) [inline purchase_open_qty/init_hook.py]: same

**Review by gurneyalex — APPROVED** [review]
Preapproving, please fix the logging messages :)

**Comment by sonhd91** [comment]
> Preapproving, please fix the logging messages :)

Oops sorry, branch update, please take a look

**Review by celm1990 — APPROVED** [review]
LGTM

**Pull request merged**